packages/integration-tests/tests/api/accounts.test.ts

```diff
@@ -2,104 +2,262 @@ import * as request from "supertest";
+        response = await helper.performGETrequest(apiRoute);
+
+        expect(response.status).toBe(200);
+        expect(response.body).toStrictEqual(expect.objectContaining({ status: "1" }));
+        expect(response.body).toStrictEqual(expect.objectContaining({ message: "OK" }));
+        expect(response.body.result.length).toBeGreaterThan(1);
+        expect(typeof response.body.result[0].blockNumber).toStrictEqual("string");
+        expect(typeof response.body.result[0].timeStamp).toStrictEqual("string");
+        expect(typeof response.body.result[0].hash).toStrictEqual("string");
+        expect(typeof response.body.result[0].nonce).toStrictEqual("string");
+        expect(typeof response.body.result[0].blockHash).toStrictEqual("string");
+        expect(typeof response.body.result[0].transactionIndex).toStrictEqual("string");
+        expect(typeof response.body.result[0].from).toStrictEqual("string");
+        expect(typeof response.body.result[0].to).toStrictEqual("string");
+        expect(typeof response.body.result[0].value).toStrictEqual("string");
+        expect(typeof response.body.result[0].tokenName).toStrictEqual("string");
+        expect(typeof response.body.result[0].tokenSymbol).toStrictEqual("string");
+        expect(typeof response.body.result[0].tokenDecimal).toStrictEqual("string");
+        expect(typeof response.body.result[0].gas).toStrictEqual("string");
+        expect(typeof response.body.result[0].gasPrice).toStrictEqual("string");
+        expect(typeof response.body.result[0].input).toStrictEqual("string");
+        expect(typeof response.body.result[0].contractAddress).toStrictEqual("string");
+        expect(typeof response.body.result[0].cumulativeGasUsed).toStrictEqual("string");
+        expect(typeof response.body.result[0].contractAddress).toBeTruthy();
+        expect(typeof response.body.result[0].gasUsed).toStrictEqual("string");
+        expect(typeof response.body.result[0].confirmations).toStrictEqual("string");
+        expect(typeof response.body.result[0].fee).toStrictEqual("string");
+        expect(typeof response.body.result[0].l1BatchNumber).toBeTruthy();
+        expect(typeof response.body.result[0].transactionType).toStrictEqual("string");
+      });
+    });
+
+    //@id1806
+    it("Verify /api?module=account&action=tokennfttx", async () => {
+      await helper.retryTestAction(async () => {
+        const blocks = await request(environment.blockExplorerAPI).get("/blocks");
+        const blockNumber = blocks.body.items[0].number;
+        const nftAddress = await helper.getStringFromFile(bufferFile + Buffer.NFTtoL2);
+        console.log(nftAddress);
+        apiRoute = `/api?module=account&action=tokennfttx&page=1&offset=10&sort=desc&endblock${blockNumber}&startblock=0&contractaddress=${nftAddress}&address=${nftAddress}`;
```

Comment:
the same with endblock${blockNumber} , do we need '=' ?

Reply:
Fixed